modules/ocf_ldap/files/ocf.schema

```diff
@@ -76,10 +76,25 @@ attributetype ( 1.3.6.1.4.1.41759.2.2.3
         SUBSTR caseIgnoreSubstringsMatch
         SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 
+# based on ipHostNumber, 1.3.6.1.1.1.1.19
+attributetype ( 1.3.6.1.4.1.41759.2.2.4
+        NAME 'ip6HostNumber'
+        DESC 'IPv6 address, as represented by RFC1884 section 2.2.2'
+        SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+
+# based on ipHost, 1.3.6.1.1.1.2.6
+objectclass ( 1.3.6.1.4.1.41759.2.1.2
+        NAME 'ip6Host'
+        DESC 'Host supporting IPv6'
+        SUP top
+        AUXILIARY
+        MUST ( ip6HostNumber $ cn )
+        MAY ( manager $ description $ l ) )
+
 objectclass ( 1.3.6.1.4.1.41759.2.1.1
         NAME 'ocfDevice'
         DESC 'Attributes for OCF hosts'
-        SUP ( ipHost $ ieee802Device $ puppetClient )
+        SUP ( ipHost $ ip6Host $ ieee802Device $ puppetClient )
         AUXILIARY
-        MUST ( cn $ ipHostNumber $ type )
+        MUST ( cn $ type )
         MAY ( environment $ dnsCname $ dnsA ) )
```

Comment:
Add `ipHostNumber` and `ip6HostNumber` here as optional attributes?

Reply:
`ipHostNumber` be mandatory IMO (which it currently is due to the hierarchy with `ipHost`). We could add `ip6HostAddress` here directly as optional, but it seems strange as described above.